test.sh

```diff
@@ -30,7 +30,21 @@ echo "Installation complete"
 rm "$PKG"
 
 install_swift() {
-  eval "$(curl -sL https://swiftenv.fuller.li/install.sh)"
+  swiftFile = ".swift-version"
+  if [ -f "$file" ]
```

Comment:
This isn't right, you should be looking for `~/.swiftenv` to determine if `swiftenv` is installed.